### PR TITLE
Clear forwarded ports for Android on process exit

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
We need to clear the local ports which we use to debug Android applications after we receive the exit event of the process.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: mobile/mobile-core/android-process-service.ts [hunk 1]
```diff
@@ -25,6 +31,21 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 
 	public mapAbstractToTcpPort(deviceIdentifier: string, appIdentifier: string, framework: string): IFuture<string> {
 		return (() => {
+			if (this._shouldAddProcessExitEventListeners) {
```
Comment (rosen-vladimirov) [inline mobile/mobile-core/android-process-service.ts]: can you extract the whole `if` to a new private method, for example `tryAttachToProcessExitSignals`

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**